test/jdk/java/security/KeyStore/PKCS12/WriteP12Test.java

```diff
@@ -129,13 +125,10 @@ public class WriteP12Test {
     private final Certificate caCert;
 
     WriteP12Test() throws CertificateException {
```

Comment:
As decode() only throws RuntimeExceptions, you don't need the throws anymore.  Also calling methods calling won't need to handle the exception either.